### ISSUE TITLE
Sometimes the cursor stucks at a random position in file list

### ISSUE THREAD

**Issue by jamadam** [trol73/trol-commander-dist#5]
Every now and then the cursor stucks at a random position in file list and need restart of whole app to repair. 

I can't tell any condition to reproduce but it happens less than 10 times a day. This have been occured since muCommander era.

trolCommander v0.9.6 on MacOS Yosemite 10.10.5

Thanks!

**Comment by reneverschoor** [comment]
I also reported this in https://github.com/trol73/mucommander/issues/89
It happens way more than 10 times a day for me.

**Comment by trol73** [comment]
Yes, I known about this issue. But I do not know how it can be guaranteed to reproduce. I have this bug is very rare.

**Comment by reneverschoor** [comment]
I also don't know how to reproduce it, apart from using trolCommander for a few hours max on my machine.
It's misbehaving right now, as we speak :(
I'd be more than happy to run e.g. debug versions with extra logging to help you find the cause.
